### DISCUSSION TITLE
howto save active OP-Mode for later use

### DISCUSSION THREAD

**Discussion by sabom2d** [edterbak/HeishaMoNR#161] [Q&A]
Hi,

as I am struggeling for weeks now and don't find a solution, I would like to ask for support.
I don't think it is rocket sience but I'm not getting there ( cause I'm node-red noob)
I need to switch the OP-MODE to DHW-Only Mode when HP switches to Defrost-Mode. This is working fine by now.
But when Defrost-Mode is switched off again, I want to get back to the former OP-Mode that was active before Defrost Mode. 
And here I'm struggeling, how can I save the old OP-Mode for use again after Defrost is done?

hope someone can help me here.

thanks in advance

ede

**Comment by edterbak** [comment]
Oh My G.... 

I just read this question in the Discussion area. I am sorry. I do not get notifications about this apparently. 
I will look into it if I can help you with this... 

But is it still required?

BTW.
Why do you want to use DHW for this purpose? 
You end up with colder water after a couple of times which requires probably more energy to get back up in temperature again.
